lib/ansible/module_utils/powershell/Ansible.ModuleUtils.Legacy.psm1

```diff
@@ -303,7 +347,7 @@ Function Get-FileChecksum($path, $algorithm = 'sha1')
             $fp.Dispose();
         }
     }
-    ElseIf (Test-Path -Path $path -PathType Container)
+    Elseif (Test-Path -Path $path -PathType Container)
```

Comment:
Minor nitpick, unless we standardize this, I would leave it as is.
This file either used `elseif` or `ElseIf`, introducing now also `Elseif` is adding to the entropy ;-)

Reply:
OK, I've decreased an entropy ;-)